sites/example-project/src/components/ui/QueryViewerSupport/QueryDataTable.svelte

```diff
@@ -83,12 +83,21 @@
       dataPage = updatedSlice
     }
 
+    function handleWheel(event){
+      // abort if scroll is in x-direction
+      if (Math.abs(event.deltaX) > Math.abs(event.deltaY)) {
+        return
+      }
+
+      event.preventDefault()
```

Comment:
Not a huge issue, but this abruptly stops a page scrolling when you are scrolling past a queryviewer table. This feels a bit unexpected as my page scroll is interrupted.
![CleanShot 2022-12-13 at 09 41 36](https://user-images.githubusercontent.com/58074498/207363578-0e862b4b-c82c-42a5-bad3-759144c77ac8.gif)

It it possible to only prevent the page scroll when the scroll is started over the queryviewer table?

Reply:
Have added a check to resume higher level scroll once first or last row is reached based on direct of scroll